lms/djangoapps/commerce/models.py

```diff
@@ -33,15 +38,36 @@ class Meta(object):
             'Specified in seconds. Enable caching by setting this to a value greater than 0.'
         )
     )
-    receipt_page = models.CharField(
-        max_length=255,
-        default='/commerce/checkout/receipt/?orderNum=',
-        help_text=_('Path to order receipt page.')
+    site = models.ForeignKey(
+        Site,
+        on_delete=models.SET_NULL,
+        blank=True,
+        null=True,
+        related_name='commerce_configuration'
```

Comment:
I guess this is the right spot to focus on in terms of related_name -- I would prefer the RN to remain 'commerceconfiguration' if possible.